src/styles/blocks/watch-center.scss

```diff
@@ -0,0 +1,7 @@
+.watch-center {
```

Comment:
Think you don't need this separate file and move it to the watch.scss file. Then this should rather be a block class like `watch__center`